todo-app/src/App.js

```diff
@@ -1,13 +1,89 @@
-import React, { Component } from 'react';
+//jshint esnext: true
+
+import React from "react";
+
+import TodoList from "./Components/TodoList";
+import AddTodoItem from "./Components/AddTodoItem";
+import { loadTodos, setTodos } from "./storage/storage";
+import "./App.css";
+
+export default class App extends React.Component {
+  state = {
+    todos: loadTodos(),
+    isEditing: null
+  };
+
+  addNewTodos = todos => {
```

Comment:
The name of this method is a bit off, as it doesn't *add* new todos, but rather replaces the list. Perhaps rename to `setTodos` or `updateTodos`?